src/bower_components/apiclient/apiclientcore.js

```diff
@@ -1195,8 +1195,11 @@ define(["events", "appStorage"], function(events, appStorage) {
         return this.ajax({
             type: "POST",
             url: url,
-            data: JSON.stringify(user),
-            contentType: "application/json"
+            data: user,
+                    Name: user.Name,
```

Comment:
Oh the Github selection didn't let me select multiple lines. Lines #1199-1201 should be removed.

Reply:
Whoops, missed that